subprojects/gradle/instrumentation-test-impact-analysis/src/main/kotlin/com/avito/instrumentation/impact/InstrumentationTestImpactAnalysisPlugin.kt

```diff
@@ -39,8 +41,10 @@ class InstrumentationTestImpactAnalysisPlugin : Plugin<Project> {
                 group = TASK_GROUP
                 description = "Analyze androidTest bytecode to collect maps: [Screen:Test], [Screen:RootId]"
 
-                //todo we should also support flavors here
-                dependsOn("${project.path}:compile${it.testBuildType.capitalize()}AndroidTestKotlin")
+                val testBuildVariant = extension.testBuildVariant.convention(it.testBuildType)
```

Comment:
I check this locally.
We need support `Flavors` in the test-project-generator for writing unit tests